testing/pom.xml

```diff
@@ -45,5 +45,20 @@
             <artifactId>spring-context</artifactId>
             <version>${spring-context.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.flywaydb</groupId>
+            <artifactId>flyway-core</artifactId>
+            <version>9.10.0</version>
```

Comment:
Pls extract the version number into `properties` like above 😄

Reply:
done